app/src/main/kotlin/com/wire/android/ui/calling/ongoing/participantsview/ParticipantsTiles.kt

```diff
@@ -117,6 +109,14 @@ fun VerticalCallingPager(
                             onDoubleTap = onDoubleTap
                         )
                     }
```

Comment:
maybe add a code comment here that the  requestVideoStreams should be called after creating renderers